setup/autoinstall-iso/docs/overview.md

```diff
@@ -0,0 +1,81 @@
+implementation.
+
+- **iso-create.md**: This guide is for *creating* a fresh ISO image from scratch. It tells you which shell
+script should be run to do this and then walks through what the creation process looks like. The guide also
+mentions what can be changed by the user in the shell script to produce a different result.
+
+- **iso-change.md**: This guide is for *updating* a previously existing iso image to apply any changes that
+were made in the iso directory. It tells you which shell script should be run to do this and then waks through
+what the shell script does to update the existing iso image.
+
+- **iso-update-release.md**: This guide is for updating the `iso-create.sh` to a different open source release
+of Ubuntu Server. The guide walks trough what to change in the shell script to achieve a different release.
+
+- **vm-setup.md**: This guide is for setting up a Virtual Machine using OracleBox to run the iso. It walks
+through all the necessary steps to ensure the Virtual Machine runs properly without error. It also walks through
+how to add a new iso to a pre-existing setup VM.
+
+### Changes to Note
+For the `user-data` file found in `/setup/autoinstall-iso/server`
+
+1. In the late commands section, **you must chose which post install script you want to run on the system once
+it finishes downloading**. By default, the post install script that runs automatically is the one found on the
+main branch of the git repository. For testing purposes, there is a development script that can be modified
+separate from the repo that helps with development. This is the second option that by default is commented out.
+Feel free to chose whichever one you need, but **only uncomment one at a time**.
+
+For the post install commands to install the required packages for the depth sensor camera:
+
+1. These are currently found in the `dev-post-install.sh` located at `/setup/autoinstall-iso/server` as they
+haven't been approved into the main branch yet. When adding the Intel RealSense repository key the Jammy update
+is the most recent update that is supported by Intel RealSense as of March 2025. The system by default runs off
+of the Noble update which is one version above the Jammy update which is why Jammy must be hardcoded into the
+repository link as Noble is not yet supported. Eventually, this should change to be a variable that grabs the
+codename for the current version and downloads the correct repo. The depth sensor camera still runs fine on the
+newer version even when using Jammy instead of Noble. The line I am referring to is:<br>
+`sudo echo "deb [signed-by=/etc/apt/keyrings/librealsense.pgp] https://librealsense.intel.com/Debian/apt-repo jammy main" | sudo tee /etc/apt/sources.list.d/librealsense.list`
+
+For the system as a whole:
+
+1. The system by fault runs off of Ubuntu Server 24.04.2. The system itself has not been fully tested for backwards
+compatability which earlier versions of Ubuntu Server. This will have to be tested.

```

Comment:
```suggestion
1. The system by default runs off of Ubuntu Server 24.04.2. The system itself has not been fully tested for backwards
compatibility with earlier versions of Ubuntu Server. This will have to be tested.

```